skeleton/iohk-skeleton.cabal

```diff
@@ -0,0 +1,157 @@
+cabal-version:       2.2
+
+-- Initial package description 'iohk-skeleton.cabal' generated by 'cabal
```

Comment:
Can we strip out the cabal generated comments? As a reader, I'd assume that any comments are from you about iohk-nix stuff.